README.md

```diff
@@ -276,6 +276,8 @@ The Hyperlight project Slack is hosted in the CNCF Slack #hyperlight. To join th
 
 For more information, please refer to our compilation of documents in the [`docs/` directory](./docs/README.md).
 
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fhyperlight-dev%2Fhyperlight-wasm.svg?type=shield&issueType=license)](https://app.fossa.com/projects/git%2Bgithub.com%2Fhyperlight-dev%2Fhyperlight-wasm?ref=badge_shield&issueType=license)
```

Comment:
IT looks like this is the badge for hyperlight-wasm not hyperlight?

Reply:
Nice catch!